deploy/finalization.go

```diff
@@ -198,7 +158,7 @@ func finalizeNode(node db.Node, details *db.DeploymentDetails, absNum int) error
 	}
 
 	_, err = client.DockerExecd(node,
-		fmt.Sprintf("sh -c 'nibbler --node-type %s --api %s --jwt %s --testnet %s --node %s %s 2>&1 >> /nibbler.log'",
-			details.Blockchain, conf.APIEndpoint, details.GetJwt(), node.TestNetID, node.ID, files))
+		fmt.Sprintf("sh -c 'nibbler --node-type %s --jwt %s --testnet %s --node %s %s 2>&1 >> /nibbler.log'",
```

Comment:
Is this the new desired docker exec command?

Reply:
We will remove nibbler as well soon

Reply:
Will that be a separate gen ticket? Or should I do it in this PR?

Reply:
Separate, I think